testsuite/MDAnalysisTests/coordinates/test_crd.py

```diff
@@ -66,11 +69,36 @@ def CRD_iter(fn):
         for ref, other in zip(CRD_iter(CRD), CRD_iter(outfile)):
             assert ref == other
 
-    def test_write_EXT(self):
-        # TODO: Write tests that use EXT output format
-        # Must have *lots* of atoms, maybe fake the system
-        # to make tests faster
-        pass
+    def test_write_EXT(self, u, outfile):
+        # Use the `extended` keyword to force the EXT format
+        u.atoms.write(outfile, extended=True)
+
+        with open(outfile, 'r') as inf:
+            format_line = inf.readlines()[2]
+            assert 'EXT' in format_line, "EXT format expected"
+
+    def test_read_EXT(self, u, outfile):
+        # Read EXT format and check atom positions
+        u.atoms.write(outfile, extended=True)
+
+        u2 = mda.Universe(outfile)
+
+        sel1 = u.select_atoms('all')
+        sel2 = u2.select_atoms('all')
+
+        # Rounding floats since EXT format support more decimals
+        cog1 = np.around(sel1.center_of_geometry(),6)
+        cog2 = np.around(sel2.center_of_geometry(),6)
+
+        assert_equal(len(u.atoms), len(u2.atoms)), 'Equal number of '\
+                'atoms expected in both CRD formats'
+        assert_equal(len(u.atoms.residues),
+            len(u2.atoms.residues)), 'Equal number of residues expected in'\
+                        'both CRD formats'
+        assert_equal(len(u.atoms.segments), 
+            len(u2.atoms.segments)), 'Equal number of segments expected in'\
+                        'both CRD formats'
+        assert_equal(cog1, cog2), 'Same centroid expected for both CRD formats'
```

Comment:
Use [`assert_allclose()`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html) with proper tol=1e-6 and atol=0